### PR TITLE
Add `handwired/dactyl_cc` keyboard

### PR DESCRIPTION
**Pull request by dunk2k**
## Description
- keyboard files added
- `default` and `via` keymaps included

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* n/a

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_cc/readme.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Dactyl-CC / Ergo S-1
+
+![DactylCC](https://i.imgur.com/CUbPLZC.jpeg)
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/readme.md:3]:
```suggestion
![DactylCC](https://i.imgur.com/CUbPLZCh.jpeg)
```

File: keyboards/handwired/dactyl_cc/readme.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Dactyl-CC / Ergo S-1
+
+![DactylCC](https://i.imgur.com/CUbPLZC.jpeg)  
+*Dactyl-CC*
+
+![ErgoS-1](https://wizardkeyboards.com/wp-content/uploads/2021/11/20211108_233645-1024x498.jpg)
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/readme.md:6]: This needs to be rehosted.

File: keyboards/handwired/dactyl_cc/readme.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* Hardware Availability: [Dactyl CC GitHub](https://github.com/mjohns/dactyl-cc)
+
+Make example for this keyboard (after setting up your build environment):
+```
+    make handwired/dactyl_cc:default
+```
+Flashing example for this keyboard:
+```
+    make handwired/dactyl_cc:default:flash
+```
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/readme.md:26]:
```suggestion

    make handwired/dactyl_cc:default

Flashing example for this keyboard:

    make handwired/dactyl_cc:default:flash

```

File: keyboards/handwired/dactyl_cc/readme.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+```
+    make handwired/ergo_s-1:default
+```
+Flashing example for this keyboard:
+```
+    make handwired/ergo_s-1:default:flash
+```
+
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/readme.md:42]:
```suggestion

    make handwired/ergo_s-1:default

Flashing example for this keyboard:

    make handwired/ergo_s-1:default:flash


```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/dactyl_cc/readme.md (hunk 2 above)
Comment (dunk2k) [reply keyboards/handwired/dactyl_cc/readme.md:6]: re-hosted as in [ErgoS-1] image URL to move from wizardkeyboards.com to an open platform such as imgur?

Comment (dunk2k) [reply keyboards/handwired/dactyl_cc/readme.md]: Changed domain of Ego S-1 image, does it resolve this conversastion?

Comment (dunk2k) [reply keyboards/handwired/dactyl_cc/readme.md]: image domain, for Ergo S-1 image, changed to imgur

File: keyboards/handwired/dactyl_cc/readme.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# Dactyl-CC / Ergo S-1
+
+![DactylCC](https://i.imgur.com/CUbPLZCh.jpeg)
+*Dactyl-CC*
+
+![ErgoS-1](https://i.imgur.com/JTLCw3N.jpg)
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/readme.md:6]:
```suggestion
![ErgoS-1](https://i.imgur.com/JTLCw3Nh.jpg)
```

File: keyboards/handwired/dactyl_cc/config.h [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+/* mouse config */
+#define MOUSEKEY_DELAY          0
+// Lowering MOUSEKEY_INTERVAL raises the cursor speed significantly, you may want to lower MOUSEKEY_MAX_SPEED (default: 10)
+#define MOUSEKEY_INTERVAL       20
+#define MOUSEKEY_MAX_SPEED       7
+#define MOUSEKEY_WHEEL_DELAY 0
+
+/* layer config */
+#define TAPPING_TOGGLE 1
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/config.h:28]: these should be defined at the keymap level

Comment (dunk2k) [reply keyboards/handwired/dactyl_cc/config.h]: Moved as per commit `60fd23381992e30f799ef39db2a5b400d45c58a7`

is the move correct?

File: data/mappings/keyboard_aliases.hjson [hunk 7]
```diff
@@ -200,6 +200,9 @@
     "gmmk/pro/iso": {
         "target": "gmmk/pro/rev1/iso"
     },
+    "handwired/ergo_s-1":{
+        "target": "handwired/dactyl_cc"
+    }
```
Comment (fauxpark) [inline data/mappings/keyboard_aliases.hjson:205]:
```suggestion
```

File: keyboards/handwired/dactyl_cc/readme.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+*`handwired/ergo_s-1` is an alias of `handwired/dactyl_cc`*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/ergo_s-1:default
+
+Flashing example for this keyboard:
+
+    make handwired/ergo_s-1:default:flash
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_cc/readme.md:39]:
```suggestion
```

File: keyboards/handwired/dactyl_cc/readme.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+![ErgoS-1](https://i.imgur.com/JTLCw3Nh.jpg)
+*Ergo S-1*
+
+Modelled on the *Kinesis Advantage* (KA) 1|2, though missing it's function and program row, board(s) retain the finger and thumb key placements of the KA but opt to psychically split the handed key-wells instead.
+
+Electronically both the Dactyl-CC and Ergo S-1 are indentical, though psychically the Ergo S-1 has larger casing designed to be used as a palm rest.
+
+* Hardware Supported: Pro micro controller or clones of
+
+## Dactyl-CC
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_cc/readme.md:15]:
```suggestion
```

File: keyboards/handwired/dactyl_cc/readme.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+![DactylCC](https://i.imgur.com/CUbPLZCh.jpeg)
+*Dactyl-CC*
+
+![ErgoS-1](https://i.imgur.com/JTLCw3Nh.jpg)
+*Ergo S-1*
+
+Modelled on the *Kinesis Advantage* (KA) 1|2, though missing it's function and program row, board(s) retain the finger and thumb key placements of the KA but opt to psychically split the handed key-wells instead.
+
+Electronically both the Dactyl-CC and Ergo S-1 are indentical, though psychically the Ergo S-1 has larger casing designed to be used as a palm rest.
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_cc/readme.md:11]:
```suggestion

Modelled on the *Kinesis Advantage* (KA) 1|2, though missing it's function and program row, board(s) retain the finger and thumb key placements of the KA but opt to psychically split the handed key-wells instead.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: data/mappings/keyboard_aliases.hjson (hunk 7 above)
Comment (dunk2k) [reply data/mappings/keyboard_aliases.hjson:205]: Can I ask as to what purpose for this change?

Would it be preferred the Ergo S-1 has it's own dir in `keyboards`?

Comment (fauxpark) [reply data/mappings/keyboard_aliases.hjson:205]: This is not what `keyboard_aliases` is for. Hyphens are also not allowed in keyboard names.

QMK does not differentiate keyboards by things like housing; per the readme, these two boards are "electronically the same", which I assume to mean it is the exact same PCB. Therefore it should have only one name in the repo.

File: keyboards/handwired/dactyl_cc/info.json [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+        "rows": ["F6", "F7", "B1", "B3", "B2", "B6"]
+    },
+      "diode_direction": "COL2ROW",
+      "split": {
+        "enabled": true,
+        "soft_serial_pin": "D0"
+      },
+    "processor": "atmega32u4",
+    "bootloader": "caterina",
+    "development_board": "promicro",
```
Comment (drashna) [inline keyboards/handwired/dactyl_cc/info.json:30]:
```suggestion
    "development_board": "promicro",
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**